parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -443,6 +396,153 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+// Add bridge pallets (GPA)
+
+/// Add granda bridge pallet to track Wococo relay chain on Rococo BridgeHub
+pub type BridgeGrandpaWococoInstance = pallet_bridge_grandpa::Instance1;
+impl pallet_bridge_grandpa::Config<BridgeGrandpaWococoInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type BridgedChain = bp_wococo::Wococo;
+	type MaxFreeMandatoryHeadersPerBlock = ConstU32<4>;
+	type HeadersToKeep = RelayChainHeadersToKeep;
+	type WeightInfo = weights::pallet_bridge_grandpa_bridge_wococo_grandpa::WeightInfo<Runtime>;
+}
+
+/// Add granda bridge pallet to track Rococo relay chain on Wococo BridgeHub
+pub type BridgeGrandpaRococoInstance = pallet_bridge_grandpa::Instance2;
+impl pallet_bridge_grandpa::Config<BridgeGrandpaRococoInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type BridgedChain = bp_rococo::Rococo;
+	type MaxFreeMandatoryHeadersPerBlock = ConstU32<4>;
+	type HeadersToKeep = RelayChainHeadersToKeep;
+	type WeightInfo = weights::pallet_bridge_grandpa_bridge_rococo_grandpa::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	pub const RelayChainHeadersToKeep: u32 = 1024;
+	pub const ParachainHeadsToKeep: u32 = 64;
+	pub const MaxRequests: u32 = 64;
```

Comment:
Is `MaxRequests` still referenced somewhere?

Reply:
good catch, it is a leftover, will be removed